test/code_size/hello_world_wasm.json

```diff
@@ -3,8 +3,8 @@
   "a.html.gz": 431,
   "a.js": 292,
   "a.js.gz": 252,
-  "a.wasm": 104,
+  "a.wasm": 103,
```

Comment:
Size has shrunk by 1 byte because `wasm-opt` doesn't strip debug/producer sections anymore, which may sound strange. The saving comes from the code section, and what I suspect is, after generating and optimizing stack IR passes, an empty function does not contain any instruction, but before we used to strip debug and producer sections with `wasm-opt` and a `nop` is added in the process of that. This does not happen anymore because we strip those sections with `llvm-objcopy`. I don't now why stripping those sections cause a `nop` to be added. I dug a little, but I'm not sure it's worth more investigation.. It's 1 byte, and it got better after all.